src/Symfony/Component/Workflow/Exception/InvalidTokenConfigurationException.php

```diff
@@ -11,10 +11,14 @@
 
 namespace Symfony\Component\Workflow\Exception;
 
+trigger_deprecation('symfony/workflow', '5.3', sprintf('The "%s" class is deprecated.', InvalidTokenConfigurationException::class));
```

Comment:
While checking the doc issue for this (https://github.com/symfony/symfony-docs/issues/14987) I found this PR. I think we should always include an alternative in the deprecation messages.

If the developer sees just this message -> _"The InvalidTokenConfigurationException class is deprecated."_ they don't know what to do. OK, it's deprecated but ... is there a new class that replaces it? Is this gone for good without an alternative?

Thanks!

Reply:
Yes it's gone. This exception is not thrown anymore by symfony.